bokeh/models/markers.py

```diff
@@ -40,19 +51,17 @@
 
 Markers are all subclasses of ``Glyph``. Additionally, they all share the
 same common interface providing fill and line properties provided by their
```

Comment:
A simple solution to prevent any confusion might be to just delete the sentence "Additionally, they all share the same [...] base class ``Marker``." This base class does not exist anymore.

Reply:
If we want to be explicit, markers accept:

* x, y position
* size in pixels
* line / fill / hatch properties

However now there is just Scatter alone, it's not a matter of different clsses sharing the same interface. But we could mentions that the above set of properties is what characterizes a "marker".

Reply:
`angle` as well.